src/app/settings.js

```diff
@@ -1,3 +1,128 @@
-export const peopleIdArray = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83];
-export const starshipsIdArray = [5,9,10,11,12,13,15,21,22,23,27,28,29,31,39,40,41,43,47,48];
-export const vehiclesIdArray = [4,6,7,8,14,16,18,19,20,24,25,26,30,33,34,35,36,37,38,42];
\ No newline at end of file
+export const peopleIdArray = [
```

Comment:
To bym chyba zostawił jak było. Dużo czytelniejsze jest w formie jednolinijkowej :-) 
Nie wiem czy specjalnie robiłaś autoformat tego pliku, czy jest jakaś komenda na autoformat całości?

Reply:
Działa to tak:
Prietterrc zmienia tylko te zacommitowane pliki.
Jeśli plik powstał przed dodaniem wtyczki to zostanie zmieniony podczas jego commitu.
Jeśli został zacommitowany (p. przez nieuwagę) plik z dodaną przykładowo jedną spacją to też zostanie sformatowany i zmieniony. Dlatego trzeba sprawdzać co commitujemy.
P.S.
PrietierRc zawsze działa tak samo. 
Dodatkowo zmienny wygląd jest, gdy przekroczymy 80 znaków w linijce - wtedy tak jak ustalilismy dochodzi do zawijania i...
... i zamiast (mniej niż 80)
function fnMniejNiż80 (arg1, arg2, ard3){}

uzyskamy (więcej niż 80)
function fnWięcejNiż80 (
arg1, 
arg2, 
ard3
){}

Ustawiłem go na wartościach domyślnych (80 znaków) tak jak uzgodniliśmy!